.github/workflows/gh-pages.yml

```diff
@@ -65,3 +65,39 @@ jobs:
         with:
           deploy_key: ${{ secrets.ACTIONS_DEPLOY_KEY }}
           publish_dir: ./public
+          
+  run-test-suite:
+    runs-on: ubuntu-18.04
+    timeout-minutes: 15
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Upgrade pip
+        run: python3 -m pip install --upgrade pip
+        
+      - name : Install pipenv
+        run : python -m pip install --upgrade pipenv wheel
+  
+      - id: cache-pipenv
+        uses: actions/cache@v1
+        with:
+          path: ~/.local/share/virtualenvs
+          key: ${{ runner.os }}-pipenv-${{ hashFiles('**/Pipfile.lock') }}
+     
+      - name: Install dependencies
+        if: steps.cache-pipenv.outputs.cache-hit != 'true'
+        run: pipenv install --deploy --dev
```

Comment:
I am not sure about the `--deploy` flag. Why is it needed?

Reply:
See above. Removed it and seems it's not needed.

Reply:
Mhh